packages/terra-layout/src/Layout.jsx

```diff
@@ -44,6 +44,14 @@ class Layout extends React.Component {
     });
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps !== prevState.prevProps) {
```

Comment:
Is this conditional ever false?

Reply:
Yes, it will evaluate to true when changing pages on the doc site but on initial render it evaluates to false